tests/unit/synthesizers/test_realtabformer.py

```diff
@@ -0,0 +1,46 @@
+"""Tests for the realtabformer module."""
+
+from unittest.mock import MagicMock, patch
+import numpy as np
+import pytest
+import pandas as pd
+from sdgym.synthesizers import RealTabFormerSynthesizer
+
+
+@pytest.fixture
+def sample_data():
+    """Provide sample data for testing."""
+    n_samples = 10
+    num_values = np.random.normal(size=n_samples)
+
+    return pd.DataFrame({
+        'num': num_values,
+    })
+
+
+class TestRealTabFormerSynthesizer:
+    """Unit tests for RealTabFormerSynthesizer integration with SDGym."""
+
+    def test_get_trained_synthesizer(self):
+        """Test _get_trained_synthesizer initializes and fits REaLTabFormer with correct parameters."""
+        data = MagicMock()
+        metadata = MagicMock()
+
+        with patch('realtabformer.REaLTabFormer') as MockREaLTabFormer:
```

Comment:
for the sake of consistency can we patch using the same format as here
https://github.com/sdv-dev/SDGym/blob/490d8c02e6e65a4c165c3766f87a237705928714/tests/unit/synthesizers/test_column.py#L9